views/recipe.pug

```diff
@@ -9,61 +9,66 @@ html(lang='en')
+                            div(class='recipe__review--content')
+                                div(class='reviewContent__name--andRating')
+                                    span(class='userName_and_rating')= `${recipeReview.User.userName} ${recipeReview.rating}`
+                                    -let n=0
+                                    while n<recipeReview.rating
+                                        span(id='the-stars') &#127775
+                                        -n++
+                                div(class='reviewContent__the--content')
+                                    p(id='review__content')=recipeReview.revContent
+                        //- how to oder the reviews
 
-            div(class='review__form')
-                form(action=`/api/recipes/${recipe.id}/review` method='post' id='reviewForm' class='hidden')
+        div(class='review__form hidden')
+            form(action=`/api/recipes/${recipe.id}/review` method='post' id='reviewForm')
+                input(type='hidden' id='recipeId' value=recipe.id)
+                label(for='revContent' class='revFormLabel') Write your review!
+                textarea(name='revContent' cols='30' row='10' class='revContent' placeholder=name)
 
-                    label(for='revContent' class='revFormLabel') Write your review!
-                    textarea(name='revContent' cols='30' row='10' class='revContent') Your review here!
-
-                    label(for='rating' class='revFormLabel') Rate the recipe
-                        select(class='ratingSelector')
-                            each val in [5, 4, 3, 2, 1]
-                                option(class='review__form--dropdown') #{val} &#127775
-                    div(class='review__form--button')
-                        button(type='submit' id='review__form--submit') Submit
-            //-    ask about if api route
-            //- div(class='addToCookbook__form')
-            //-         form(action=`/api/cookbook/${cookBook.id}/recipes/${recipe.id}` id='addToCookbookForm' class='hidden')
-            //-             h1(class='addToCookbookLabel')
-            //-             label(for='addToCookbookForm' class='addToCookBookLabel') Add this recipe to one of you cookbooks!
-            //-                     //- do we want to do a selector?
-            //-             select(class='cookbookSelector') Select A Cookbook
-            //-                 each cookbook in cookbooks
-            //-                     option(class='')
+                label(for='rating' class='revFormLabel') Rate the recipe
+                    select(class='ratingSelector' name='rating')
+                        each val in [5, 4, 3, 2, 1]
+                            option(class='review__form--dropdown') #{val} &#127775
+                div(class='review__form--button')
+                    button(type='submit' id='review__form--submit') Submit
+        //-    ask about if api route
```

Comment:
If you want to leave this as a TODO, please be sure you are referencing a specific github issue here as well.

Reply:
Removed, api route status confirmed